image/templates/common/setup-scanner-v4.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Sets up Central to use Scanner-v4. Note that deploying Scanner is a prerequisite to deploying Scanner-v4.
```

Comment:
I believe the official way to write it (nowadays) is: `Scanner V4`.

Why is it a prerequisite?

Reply:
Central expects Scanner V2 to be present, when SCANNER_V4 is enabled, right? But that wouldn't cause the script to fail. However, it does requires Central to be present because it patches the deployment. This script also doesn't attempt to deploy the pull secret (as `setup-central.sh` and `setup-scanner.sh` do), because it expects that at least one of them was executed previously. 

Should I rephrase it so that it states that it requires Central, or is it self-explanatory (because "it sets up Central...")?

Reply:
> Central expects Scanner V2 to be present, when SCANNER_V4 is enabled, right?

tbh, I don't know. I am sure @RTann or @dcaravel can comment on this.

Regarding the pull secrets, I think I understand your point, but this deserves another thought. I am not sure about our assumptions regarding the image registries here (especially for Scanner V4 in contrast to Scanner V2).

> Should I rephrase it so that it states that it requires Central, or is it self-explanatory (because "it sets up Central...")?

Personally I don't think that you need to add such a remark. Generally speaking, could it be the case that the same snippets here are also used for generating the README of scanner deployment bundles? If so, this is something we should keep in mind when changing the language here.

Reply:
> > Central expects Scanner V2 to be present, when SCANNER_V4 is enabled, right?
> 
> tbh, I don't know. I am sure @RTann or @dcaravel can comment on this.

Yes we expect Scanner V2 to be running alongside Scanner V4 until the features in Scanner V2 (node scanning) are ported over to Scanner V4.  That being said, it is 'technically' possible to not install Scanner V2 and only install Scanner V4 (some features, such as node scanning, would not work then) -  I would call that an 'unsupported setup' however as it has not been tested - so the comment calling scanner v2 a pre-req for scanner v4 is OK by me.